app/demo-geant-integration/SensitiveDetector.hh

```diff
@@ -19,11 +25,21 @@ namespace demo_geant
 class SensitiveDetector final : public G4VSensitiveDetector
 {
   public:
-    explicit SensitiveDetector(std::string name);
+    //!@{
+    //! \name Type aliases
+    using SensitiveHitsCollection = G4THitsCollection<SensitiveHit>;
```

Comment:
Since this isn't part of the public interface it should be private with the data.

Reply:
Actually it will be used outside this class when hit collections are retrieved (for example, at the end of Event or Run).
Anyway, I dropped public and will put it back when it is necessary.